docs/pages/ds-compositeIn-L3-Low-Level-TestSpec.md

```diff
@@ -0,0 +1,41 @@
+# Device Settings Composite Input L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Composite Input module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `Y`   \- yes supported
+- `NA`  \- Not Supported
+
+### References
+
+- `High Level Test Specification` - [ds-compositeIn-High-Level_TestSpec.md](https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/docs/pages/ds-compositeIn-High-Level_TestSpec.md)
+
+- `HAL Interface file` - [dsCompositeIn Header 4.0.0](https://github.com/rdkcentral/rdk-halif-device_settings/blob/4.0.0/include/dsCompositeIn.h)
+
+
+## Level 3 Test Cases High Level Overview
+
+Below are top test use-case for the compositeIn.
+
+|#|Test-case|Description|Focus APIs|Source|Sink|
```

Comment:
remove sink & source columns, it makes no sence for this component.

The component is called Composite Input.

Reply:
done